test/functional/test-action.js

```diff
@@ -544,6 +663,64 @@ describe('Action method', () => {
 });
 
 
+describe('Multiple handlers action method', () => {
+  let sandbox;
+  let win;
+  let action;
+  let onEnqueue1, onEnqueue2;
+  let targetElement, parent, child, execElement1, execElement2;
+
+  function createExecElement(id, enqueAction) {
+    const execElement = document.createElement('amp-element');
+    execElement.setAttribute('id', id);
+    execElement.enqueAction = enqueAction;
+    return execElement;
+  }
+
+  beforeEach(() => {
+    sandbox = sinon.sandbox.create();
+    win = {
+      document: {body: {}},
+      services: {
+        vsync: {obj: {}},
+      },
+    };
+    action = new ActionService(new AmpDocSingle(win), document);
+    onEnqueue1 = sandbox.spy();
```

Comment:
I changed some names in the mixed test case since I thought it would benefit the most.